activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+*   `assert_difference` message now includes what changed.
+
+    This makes it easier to debug non-obvious failures.
+
+    Before:
+
+    ```
+    "User.count" didn't change by 32.
+    Expected: 1611
+      Actual: 1579
+    ```
+
+    After:
+
+    ```
+    "User.count" didn't change by 32 (changed by 0).
```

Comment:
```suggestion
    "User.count" didn't change by 32 but by 0.
```

I think that's how I'd do it, but I'm far from a good English speaker.

Reply:
I like that!